internal/utils/kruize/kruize_api.go

```diff
@@ -196,3 +196,32 @@ func Is_valid_recommendation(recommendation kruizePayload.Recommendation, experi
 		return false
 	}
 }
+
+func Delete_experiment_from_kruize(experiment_name string) {
+	url := cfg.KruizeUrl + "/createExperiment"
+	data := []map[string]string{
+		{"experiment_name": experiment_name},
+	}
+	payload, err := json.Marshal(data)
+	if err != nil {
+		log.Error("unable to marshal payload to json: ", err)
+		return
+	}
+
+	client := &http.Client{}
+	req, err := http.NewRequest("DELETE", url, bytes.NewBuffer(payload))
+	if err != nil {
+		log.Error("unable to create request object ", err)
```

Comment:
same here, we should increment counter and err about deletion,
~~~
kruizeAPIException.WithLabelValues("/deleteExperiment").Inc()
log.Error("error Occured while deleting experiment: ", err)
~~~

Reply:
I will add the counter and change the error message.